Example/Stripe iOS Example (Custom)/ViewController.m

```diff
@@ -142,7 +142,9 @@ - (IBAction)beginStripeCheckout:(id)sender {
     STPCheckoutOptions *options = [[STPCheckoutOptions alloc] initWithPublishableKey:[Stripe defaultPublishableKey]];
     options.purchaseDescription = @"Cool Shirt";
     options.purchaseAmount = 1000; // this is in cents
+    options.requireBillingAddress = @YES;
     options.logoColor = [UIColor purpleColor];
+    options.logoImage = [UIImage imageNamed:@"dribbble"];
```

Comment:
Agh, sorry - I mistakenly committed a few changes that I was using for local testing. (crazyidea: I should have some kind of gitignored local overrides file for this). This shouldn't be here.